src/clj/game/core/diffs.clj

```diff
@@ -54,6 +55,13 @@
              :source-info {:ability-idx ability-idx}}]
     (if (and (or (active? card)
                  (:autoresolve ability))
+             ;; using the reg instead of any-effect here because this function gets called
+             ;; a LOT, and this is very close to linear time and should (I think)
+             ;; always be correct when this fn is called.
+             ;; If that's ever an issue, we can switch this to:
+             ;; (is-disabled? state side card)
+             ;; --n kelly, apr 2024
+             (not (is-disabled-reg? state card))
```

Comment:
Diffs are generated after `fake-checkpoint` is called, so it'll be up-to-date. You can leave the comment if you want, but you don't need to change it to `is-disabled?`.

Reply:
yeah, I think that comment is probably worth keeping for now. No clue what changes we'll make in the next decade :joy:.